### ISSUE TITLE
Still crashes

### ISSUE THREAD

**Issue by evyboy1** [exelix11/SwitchThemeInjector#60]
i use sx os for homebrew and i am trying to use nxthemes the themes dont install i don't know why but it needs a fix.

**Comment by evyboy1** [comment]
the theme does not install even when it works

**Comment by exelix11** [comment]
#57 possibly? For stuff not installing create a folder called `sxos` in the root of your sd.

**Comment by evyboy1** [comment]
ok i will do that

**Comment by evyboy1** [comment]
i did that but it still crashes

**Comment by evyboy1** [comment]
omg it worked thxx

**Comment by evyboy1** [comment]
most of the themes crash my switch

**Comment by evyboy1** [comment]
i am mad

**Comment by evyboy1** [comment]
but some of my themes work

**Comment by exelix11** [comment]
Console firmware ? 5.0 up to 10.1.0 are supported.

It's probably sd corruption on your end, I'd say format your sd to FAT32, copy back everything and try again

**Comment by exelix11** [comment]
Also notably, are you installing szs or nxtheme files? Szs files are firmware specific and won't work on different firmware versions. Nxthemes should work.

To be clear, where do you get the crash? On boot or while installing a theme?

**Comment by evyboy1** [comment]
instailling a theme

**Comment by evyboy1** [comment]
and its nxtheme

**Comment by evyboy1** [comment]
my sd card is exfat is it supported?

**Comment by exelix11** [comment]
Yes but exfat is known to cause issues due to corruption. First thing you can try is completely deleting the themes folder and copying your files again. But I'd recommend switching to fat32

**Comment by evyboy1** [comment]
i will do it

**Comment by evyboy1** [comment]
i am copying my files because they are inportint

**Comment by evyboy1** [comment]
it cant be fat32

**Comment by exelix11** [comment]
Windows doesn't show the option for bigger drives but there's a program to do it anyway, don't remember the name but there are several guides on the internet.

**Comment by evyboy1** [comment]
i have a 128 gb sd ard

**Comment by 69NIGGER420** [comment]
Lol gay

**Comment by 69NIGGER420** [comment]
Hi exelix

**Comment by 69NIGGER420** [comment]
Nou

**Comment by 69NIGGER420** [comment]
Get a life, exelix

**Comment by 69NIGGER420** [comment]
Social life

**Comment by 69NIGGER420** [comment]
Better than the shit you do

**Comment by 69NIGGER420** [comment]
Not my problem?

**Comment by 69NIGGER420** [comment]
Even if you can, I could easily make more accounts

**Comment by 69NIGGER420** [comment]
Funny part is, you'll never know which discord user I am :dogekek:

**Comment by 69NIGGER420** [comment]
Good luck coding your way through this 😎

**Comment by 69NIGGER420** [comment]
Lol coward
